src/ADAL.PCL.WinRT/WebUI.cs

```diff
@@ -73,19 +73,33 @@ public WebUI(IPlatformParameters parameters)
             {
                 if (ssoMode)
                 {
+                    string requestId = Telemetry.GetInstance().CreateRequestId();
+                    Telemetry.GetInstance().StartEvent(requestId, "ui_event");
+
+                    UIEvent UiEvent = new UIEvent();
```

Comment:
you are duplicating the code for start and stop. This could be outside the if condition